lib/inquisition/i18n_tasks/runner.rb

```diff
@@ -0,0 +1,37 @@
+require 'i18n/tasks'
+
+module Inquisition
+  module I18nTasks
+    class Runner < ::Inquisition::Runner
+      def call
+        base_task = ::I18n::Tasks::BaseTask.new
+        parse_data_errors([base_task.missing_keys, base_task.unused_keys])
+        @issues
+      end
+
+      private
+
+      def parse_data_errors(errors)
```

Comment:
Maybe it would be better to create separate classes for processing each data type?